packages/design-system/src/components/SelectField/SelectField.md

```diff
@@ -1,3 +1,7 @@
+```js noeditor
+<Banner size="large" text="This component can be used only in React applications." type="warning"/>
```

Comment:
needs to be adjusted to use `children` instead of `text`

Reply:
```suggestion
<Banner size="large" type="warning">This component can be used only in React applications.</Banner>
```

Reply:
✅